DEPS

```diff
@@ -14,7 +14,7 @@ deps = {
   "vendor/bat-native-tweetnacl": "https://github.com/brave-intl/bat-native-tweetnacl.git@800f9d40b7409239ff192e0be634764e747c7a75",
   "vendor/challenge_bypass_ristretto_ffi": "https://github.com/brave-intl/challenge-bypass-ristretto-ffi.git@f2eff7aca4ea04564e3647b93eb72f33ebdbf683",
   "vendor/gn-project-generators": "https://github.com/brave/gn-project-generators.git@b76e14b162aa0ce40f11920ec94bfc12da29e5d0",
-  "vendor/web-discovery-project": "https://github.com/brave/web-discovery-project@763107d1d37cbe9acfccabd550148728e8de5640",
+  "vendor/web-discovery-project": "https://github.com/petemill/web-discovery-project@8fdfd45b91900b0cdd7ea063f17b109c9eb0c787",
```

Comment:
this will be changed back once (or if) https://github.com/brave/web-discovery-project/pull/70 is accepted